src/glwidget.cpp

```diff
@@ -295,14 +295,14 @@ void GLWidget::paintGL()
         ox = pRID->x;
```

Comment:
I think I have refactored this code locally in the past and found that `ox`, `oy` and `oz` are surplus and can be factored out!